lab2.js

```diff
@@ -122,24 +181,49 @@ assert(human.sayHello(klingon) === 'nuqneH',
 //*********************************************************
 function max(array) {
   // TODO: return the largest number in the given array
+  return Math.max.apply(Math, array);
 }
 
 // TODO: write three more assertions
 assert(max([ 1, 3, 2 ]) === 3, '[1,3,2]');
+assert(max([ 1, 3, 15 ]) === 15, '[1,3,15]');
+assert(max([ 60, 3, 2 ]) === 60, '[60,3,2]');
+assert(max([ 1, 18, 2 ]) === 18, '[1,18,2]');
 
-function variablify(string) {
-  // TODO: you are given a string with several words in it
+// TODO: you are given a string with several words in it
   // return a corresponding variable name that follows
   // javascript conventions
   // HINT:
   // you might want to use these string methods:
   //  split(), charAt(), toUpperCase()
   // and this array method: join()
+
+function variablify(string) {
+  var splitString = string.split(' ');
+  var value = [];
+  var i = 0;
+
+  for (i = 0, i < splitString; i++) {
```

Comment:
you have a syntax error here that's killing your code at this point. Make sure to run your file frequently to test your code - e.g. 
node lab2.js